app/views/ansible_credential/edit.html.haml

```diff
@@ -1,2 +1,2 @@
 #main_div
-  = render :partial => 'credential_form', :locals => {:newRecord => false}
+  = react 'AnsibleCredentialsForm', :recordId => @id
```

Comment:
The `@id` here can be a huge integer that if getting into JS might become a floating point number which will have precision issues, so it's better to send it as string:
```haml
= react 'AnsibleCredentialsForm', :recordId => @id.to_s
```